packages/contracts/package.json

```
@@ -0,0 +1,63 @@
+{
+    "name": "extension-contracts",
+    "description": "Semaphore extensions Contracts to manage anonymous voting and whistleblowing",
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "dev": "hardhat node & yarn deploy --network localhost",
+        "compile": "hardhat compile",
+        "deploy": "yarn deploy:semaphore-voting && yarn deploy:semaphore-whistleblowing",
+        "deploy:semaphore-voting": "yarn compile && hardhat deploy:semaphore-voting",
+        "deploy:semaphore-whistleblowing": "hardhat compile && hardhat deploy:semaphore-whistleblowing",
+        "verify": "hardhat verify",
+        "test": "hardhat test",
+        "test:report-gas": "REPORT_GAS=true hardhat test",
+        "test:coverage": "hardhat coverage",
+        "typechain": "hardhat typechain",
+        "prepublish": "tar -czf files.tgz .gitignore .yarn .yarnrc.yml",
+        "lint": "eslint . --ext .js,.ts && solhint 'contracts/**/*.sol'",
```

```suggestion
        "lint": "solhint 'contracts/**/*.sol'",
```

eslint needs to be run in the root folder